FCCSW_ecal/compute_resolutions.py

```diff
@@ -18,18 +18,20 @@ def main():
             help = "Output CSV file", type=str)
     parser.add_argument("--clusters", nargs='+', action="extend", default=[],
             help = "Cluster collections to use", type=str)
-    parser.add_argument("--MVAcalib", help = "Path to XGBoost json file used to calibrate CaloClusters",
+    parser.add_argument("--MVAcalibCalo", help = "Path to XGBoost json file used to calibrate CaloClusters",
+    type=str)
+    parser.add_argument("--MVAcalibTopo", help = "Path to XGBoost json file used to calibrate CaloTopoClusters",
     type=str)
     args = parser.parse_args()
-    run(args.inputDir, args.clusters, args.outFile, args.MVAcalib)
+    run(args.inputDir, args.clusters, args.outFile, args.MVAcalibCalo, args.MVAcalibTopo)
 
 
-def run(in_directory, clusters_colls, out_file, MVAcalib):
+def run(in_directory, clusters_colls, out_file, MVAcalibCalo, MVAcalibTopo):
     """Actual processing"""
     init_stuff()
     res = []
-    if(MVAcalib):
-        res += get_MVAcalib_resolution(in_directory, MVAcalib)
+    if(MVAcalibCalo or MVAcalibTopo):
+        res += get_MVAcalib_resolution(in_directory, MVAcalibCalo, MVAcalibTopo)
```

Comment:
In terms of code, it will be much cleaner to change get_MVAcalib_resolution back to get only 1 calibration at a time, but that the user can change, i.e:

``` python
if MVAcalibCalo:
  res += get_MVAcalib_resolution(in_directory, "CaloClusters", MVAcalibCalo)
if MVAcalibTopo:
  res += get_MVAcalib_resolution(in_directory, "CaloTopoClusters", MVAcalibTopo)
```

and have `get_MVAcalib_resolution` be:

``` python
def get_MVAcalib_resolution(in_directory, clusters, MVAcalib_file):
```

Reply:
Yes it's much cleaner. I have changed the code, thanks @gartrog.